tracking-templates/iframe-loader.js

```diff
@@ -82,13 +89,19 @@
                 message('sid', function(r) {cb && cb(r)});
             };
             g.switchChannel = function(id, r, d, cb, cb_err) {
-                message('cid;' + id + ';' + r + ';' + d, function(r) {cb && cb(r === '1')}, cb_err);
+                message('cid;' + id + ';' + r + ';' + d, function(r) {
+                    cb && cb(r === '1');
+                    setTimeout(g._sendMeta, 1);
```

Comment:
can we guarantee that at this time the session id (retrieved via `window['{{TRACKING_GLOBAL_OBJECT}}'].getSID()` has already changed

Reply:
As the callback for the `switchChannel` method is called from within the script which updates the session id (https://github.com/rbredtech/tv-insight-hbbtv-tracking-script/blob/develop/tracking-templates/new_session.js#L26), we can be sure that the session id is already updated when `_sendMeta` is called.